db.js

```diff
@@ -1,162 +1,49 @@
-(function () {
```

Comment:
Removed the IIFE as node module always wrapped inside a IIFE, hence not pollute the global namespace.